### ISSUE TITLE
Radio Button Selection not Visible 

### ISSUE THREAD

**Issue by aradhikaIbm** [IBM/carbon-components-angular#88]
## Detailed description

> Describe in detail the issue you're having.
During our AVT 2, discovered our radio buttons does not show selection when is no color setting. I'm investigating the widget library as well and noticed the same issue. 

> Is this a feature request (new component, new icon), a bug, or a general issue?
bug

> Is this issue related to a specific component?
radio button

> What did you expect to happen? What happened instead? What would you like to see changed?
The radio button should look selected. 

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?
IBM Watson Recruitment, Accessibility deadline of Dec 10, 2018.

## Steps to reproduce the issue

1. Set FF in color override setting. 
Firefox Version: 60.1.0esr (64-bit)

Firefox > Tools > Options… > Content > Fonts & Colors: Colors… >
(check/uncheck) Use system colors 
Override the colors specified by the page… = Always > OK > OK

2. View the radio buttons in the story book.

**Comment by cal-smith** [comment]
We'll want to check against the vanilla carbon components as well ... this may be something we can solve with an upstream patch

**Comment by ninja511** [comment]
Moving to Carbon repo and closing this one.
https://github.com/IBM/carbon-components/issues/1133
